buildSrc/src/main/kotlin/otel.errorprone-conventions.gradle.kts

```diff
@@ -50,7 +50,6 @@ tasks {
         disable("UnnecessarilyFullyQualified")
 
         // We use animal sniffer
-        disable("Java7ApiChecker")
```

Comment:
Apparently this was removed in a minor version. Can find a commit [here](https://github.com/google/error-prone/compare/v2.37.0...v2.38.0) in the diff, but its not mentioned in the [release notes](https://redirect.github.com/google/error-prone/compare/v2.37.0...v2.38.0). 🤷

Reply:
if you haven't already, you can check contrib and instrumentation to see what @laurit did

Reply:
Looks like we reached the same conclusions: https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1859/files